### DISCUSSION TITLE
Suggestion: Improve scrollbars style

### DISCUSSION THREAD

**Discussion by DannyBen** [retypeapp/retype#107] [Ideas]
One thing that bothers me a little with Retype sites, is the fact that the scrollbars - both for long side navigation bars, and for source code snippets (horizontal scroll) are using the standard browser styling.

This problem is more obvious with Retype than previous doc engines I have used, since the main page area is narrower, due to the double sidebars (so code snippets usually have scrollbars).

Now, I know that some developers try to avoid changing the scrollbar style for some reason, but in the same way we no longer see any standard form elements out there, I think it is time to not be afraid to apply a more gentle and modern style to scrollbars.

The need for this - in my view - is even more noticeable in dark themes, but not only.

Here is a screenshot from the Retype site
![image](https://user-images.githubusercontent.com/2405099/126905730-bc635276-089f-4987-99f3-981a199a1860.png)

And here it is with some minimal styling applied:
![image](https://user-images.githubusercontent.com/2405099/126905736-78c063e2-9ae2-4cf0-8e04-f8e3f91ef37d.png)

This (or a similar style) makes it more in line with how it looks on modern mobile devices, and tells the desktop users that although the world is "mobile first", we still care about desktop users.

Also if someone claims that this somewhat reduces usability due to the bars being narrower, I say that this is a reasonable compromise, especially considering scrolling with the mouse wheel and arrow keys is easier anyhow.

This is the CSS I used:

```css
::-webkit-scrollbar {
  width: 6px;
  height: 6px;
}
::-webkit-scrollbar-thumb {
  background: #517EFB;
  border-radius: 3px;
}
```

P.S. I would also eradicate all the white-background badges, but that's a different story that I already voiced my opinion on.

**Comment by fabriciomurta** [comment]
This scrollbar issue was something we were aware and probably only is still there because nobody raised the subject yet. Besides, the scrollbars in mac are already good as they are, so if one is navigating on the Apple system, it shouldn't bother much.

Thanks for the CSS styling suggestion by the way.

**Reply by DannyBen** [reply]
Yes... unfortunately windows scrollbar and form elements stayed in the late 90s

**Comment by geoffreymcgill** [comment]
We're going to take a crack at improving the styling of the scrollbars on Windows.

MacOS will remain with native styles. At least that's the goal.

**Reply by DannyBen** [reply]
> MacOS will remain with native styles. At least that's the goal.

Interesting. I would be curious to see if the goal can be achieved.

**Comment by red0888** [comment]
Hello, I'm still eagerly waiting for this update! Any progress yet?

**Comment by fabriciomurta** [comment]
Hello! I see this is something looked after, and in fact we do have something on the works regarding the scrollbars. We just couldn't test it enough to feel safe to include in this version, but it's not like we forgot about the issue. We actually have a pull request blinking on our eyes everyday so that we make it into the next release. No promises, though, we're not merging it until it is well integrated.

But your comments here really helps us focus on it or something else, so keep them coming!

**Comment by geoffreymcgill** [comment]
Scrollbars on Windows have been re-restyled as of the `v1.11.0` [release](https://github.com/retypeapp/retype/releases/tag/v1.11.0).

**Reply by DannyBen** [reply]
Thank you! Upgrading.
The new style is much nicer and cleaner, well done.

**Comment by red0888** [comment]
Ty!
